see https://bugs.gentoo.org/945794
https://github.com/gtkwave/gtkwave/pull/455.patch
apply same ifdef as main.c
the getopt files have been deleted in the dev branch
--- a/src/globals.c
+++ b/src/globals.c
@@ -18,7 +18,11 @@
 #include "fgetdynamic.h"
 #include "ghw.h"
 #include "globals.h"
+#ifdef HAVE_GETOPT_LONG
+#include <getopt.h>
+#else
 #include "gnu-getopt.h"
+#endif
 #include "gnu_regex.h"
 #include "gtk23compat.h"
 #include "lx2.h"
